### PR TITLE
removed 'public' from beta mentions

### PR DESCRIPTION
**Pull request by MarkWoulfeNeo4j**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-aura-236.surge.sh

**Comment by MarkWoulfeNeo4j** [comment]
[Aura API: Remove term Public Beta from Docs](https://trello.com/c/fieZSPY8/188-aura-api-remove-term-public-beta-from-docs)

File: modules/ROOT/pages/platform/api/authentication.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@ label:AuraDS-Enterprise[]
 
 [TIP]
 ====
-The Aura API is currently in Public Beta for Aura Enterprise customers only. If you wish to participate in the Public Beta, please contact your Customer Success Manager.
+The Aura API is currently in beta for Aura Enterprise customers only. If you wish to participate in the beta, please contact your Customer Success Manager.
```
Comment (nvitucci) [inline modules/ROOT/pages/platform/api/authentication.adoc:10]: I wonder if we should add "participate in the beta _testing_"?

Comment (MarkWoulfeNeo4j) [reply modules/ROOT/pages/platform/api/authentication.adoc:10]: @Paul-Blewett are you happy to mention the term test here?

Comment (nvitucci) [reply modules/ROOT/pages/platform/api/authentication.adoc:10]: Or we could say _beta phase_ or _beta stage_ even.

Comment (MarkWoulfeNeo4j) [reply modules/ROOT/pages/platform/api/authentication.adoc:10]: i like stage so will go with that

Comment (MarkWoulfeNeo4j) [inline modules/ROOT/pages/platform/api/authentication.adoc:10]:
```suggestion
The Aura API is currently in beta for Aura Enterprise customers only. If you wish to participate in the beta stage, please contact your Customer Success Manager.
```

File: modules/ROOT/pages/platform/api/overview.adoc [hunk 2]
```diff
@@ -7,7 +7,7 @@ label:AuraDS-Enterprise[]
 
 [TIP]
 ====
-The Aura API is currently in Public Beta for Aura Enterprise customers only. If you wish to participate in the Public Beta, please contact your Customer Success Manager.
+The Aura API is currently in beta for Aura Enterprise customers only. If you wish to participate in the beta, please contact your Customer Success Manager.
```
Comment (MarkWoulfeNeo4j) [inline modules/ROOT/pages/platform/api/overview.adoc:10]:
```suggestion
The Aura API is currently in beta for Aura Enterprise customers only. If you wish to participate in the beta stage, please contact your Customer Success Manager.
```

**Review by nvitucci — APPROVED** [review]

**Pull request merged**